core/domain/feedback_services.py

```diff
@@ -261,3 +263,40 @@ def enqueue_feedback_message_email_task(user_id):
     taskqueue_services.enqueue_task(
         feconf.FEEDBACK_MESSAGE_EMAIL_HANDLER_URL, {'user_id': user_id},
         feconf.DEFAULT_FEEDBACK_MESSAGE_EMAIL_COUNTDOWN_SECS)
+
+
+def get_feedback_message_references(user_id):
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id, strict=True)
+
+    return [feedback_domain.FeedbackMessageReferences(
+        reference['exploration_id'], reference['thread_id'],
+        reference['message_id'])
+            for reference in model.feedback_message_references]
+
+def update_feedback_message_reference(user_id, reference):
```

Comment:
I think a better name is add_feedback_message_reference. You're updating the _model_ (and in fact doing that only sometimes), but in both cases you're basically adding a new feedback message reference to the model.